api/Makefile

```diff
@@ -84,6 +84,10 @@ gen-files .generate_files: lint-cache-dir clean-generated
 	touch .generate_files
 	$(MAKE) fix
 
+fix:
```

Comment:
Why do we need a distinct fix target her instead of using the common one? Might be worth a comment at least since this seems like an exceptional case.

Reply:
The new scripts get broken by extracting the API into its own repo (and the builds of the API deliberately _don't_ mount the API into the build container).  I thought about copying the scripts over but it seemed quite messy to duplicate them all.